src/html.sortable.js

```diff
@@ -256,6 +275,13 @@ var _listsConnected = function (curList, destList) {
   }
   return false
 }
+/*
+ * Is Copy Active for sortable
+ * @param {Element} sortable a single sortable
+ */
+var _isCopyActive = function (sortable) {
+  return _data(sortable, 'opts').copy
```

Comment:
This would need to be `return _data(sortable, 'opts').copy === true` because otherwise e.g. setting copy to 'no' would also evaluate to true.